core/domain/user_jobs_continuous.py

```diff
@@ -346,19 +349,50 @@ def map(item):
             item.id)
         contributors = exploration_summary.contributors_summary
         total_commits = sum(contributors.itervalues())
-        if total_commits == 0:
-            return
         for contrib_id in contributors:
             # Find fractional contribution for each contributor
-            contribution = contributors[contrib_id] / float(total_commits)
+            contribution = (
+                contributors[contrib_id] / float(total_commits)
+                if total_commits else 0)
             # Find score for this specific exploration
-            exploration_impact_score = value_per_user * reach * contribution
-            yield (contrib_id, exploration_impact_score)
+            exploration_impact_score = (value_per_user * reach * contribution
+                                        if value_per_user > 0 else 0)
+            exploration_data = {
+                'exploration_impact_score': exploration_impact_score
+            }
+            # if the user is an owner for the exploration, then update dict with
+            # 'average ratings' and 'total plays' as well.
+            if contrib_id in exploration_summary.owner_ids:
+                # Get num starts (total plays) for the exploration
+                num_starts = statistics['start_exploration_count']
+                exploration_data.update({
+                    'exploration_total_plays': num_starts,
+                    'exploration_average_rating': average_rating
```

Comment:
Actually I think this should be None if there are no ratings, and it should just not be counted in the calculation. What do you think?

"No ratings" is different from "rating is 0".